modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/DocumentResourceImpl.java

```diff
@@ -689,6 +691,13 @@ private ServiceContext _createServiceContext(
 		return serviceContext;
 	}
 
+	private long[] _getAncestorSiteAndDepotGroupIds(long groupId)
+		throws PortalException {
+
+		return _siteConnectedGroupGroupProvider.getAncestorSiteAndDepotGroupIds(
+			groupId, true);
+	}
+
```

Comment:
Can you inline this method? And sorry for the constant back and forth :bowing_man:

Reply:
I've inlined it. No worries about the back and forth. I squashed the last 3 commits into a single SF commit and for pushed.